kogito-usertasks-with-security-oidc-springboot/src/main/java/org/kie/kogito/tests/LogEventPublisher.java

```diff
@@ -0,0 +1,31 @@
+package org.kie.kogito.tests;
+
+import java.util.Collection;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.kie.kogito.event.DataEvent;
+import org.kie.kogito.event.EventPublisher;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LogEventPublisher implements EventPublisher {
```

Comment:
not really needed ... sort of left over after my tests so can be removed

Reply:
Thanks @mswiderski, removed! .. but now you mention the test.. They are broken regarding the keycloak server needed.. I suppose we need something similar to the KeycloakServerTestResource what we had in quarkus..Do you think makes sense to have the complete testing stuf in this use case?